packages/live-share-inking/src/core/LiveCanvas.ts

```diff
@@ -259,7 +259,7 @@ class BuiltInLiveCursor extends LiveCursor {
         }
 
         const template = document.createElement("template");
-        template.innerHTML = visualTemplate;
+        template["innerHTML"] = visualTemplate;
```

Comment:
Just curious why this change is necessary?

Reply:
Enabling Teams Build requires Guardian's team's SDL pipeline running and that enforces security rules like "no-inner-html" as modifying HTML DOM directly is a security violation.

![image](https://user-images.githubusercontent.com/5934356/190052230-119be511-1cc2-446c-a67c-423a8b688374.png)